### PR TITLE
chore: update e2e test

### PR DESCRIPTION
**Pull request by xiaolang124**
1. Update e2e test to support multi environment.
2. Delete some APIM validate codes. (Because bicep provision will change config data)
3. Do not open url in e2e test.

### CODE REVIEW

**Comment by xiaolang124** [comment]
E2E TEST: https://github.com/OfficeDev/TeamsFx/actions/runs/1488427981

**Review by hund030 — DISMISSED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Review by XiaofuHuang — DISMISSED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

File: packages/cli/tests/e2e/bot/ProvisionResourceProjectWithNewBot.tests.ts [hunk 1]
```diff
@@ -53,17 +60,34 @@ describe("Provision", function () {
...
-      const context = await readContext(projectPath);
-
-      // Validate Aad App
-      const aad = AadValidator.init(context, false, AppStudioLogin);
-      await AadValidator.validate(aad);
-
-      // Validate Bot Provision
-      const bot = BotValidator.init(context);
-      await BotValidator.validateProvision(bot);
+      if (isMultiEnvEnabled()) {
```
Comment (IvanJobs) [inline packages/cli/tests/e2e/bot/ProvisionResourceProjectWithNewBot.tests.ts:63]: Would it be better to separate multi-env cases from the original ones in a separated ts file? I mean, I saw if/else style here and concerned that can we cover both cases?

Comment (xiaolang124) [reply packages/cli/tests/e2e/bot/ProvisionResourceProjectWithNewBot.tests.ts:63]: In the end, normal case will be deleted. The only multi env case will be saved.

**Review by KennethBWSong — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
lgtm for the changes of bot

**Review by XiaofuHuang — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by LongOddCode — APPROVED** [review]

**Pull request merged**